blocks.md

```diff
@@ -190,6 +192,29 @@ Placeholder | Description
 
 ###### [↥ back to top](#list-of-available-blocks)
 
+## Bindingmode
+Creates a block which shows the binding mode in i3/sway compatible window managers. That is for example, when you press `$mod+r` to start resizing windows.
+
+When no special mode is active, i.e. the state is `default`, nothing will be shown. If another mode is triggered, the block will show a button with the name of the mode. Clicking on the button will reset the mode to `default`.
```

Comment:
By button I assume you mean button widget?

Reply:
Yes. Should I change that to read button widget?